### PR TITLE
Update mindsets.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Formatting changes, punctuation fixes, changing some lists into list format, changing some list formats from numbers to bullets. Reviewing heading capitalization. Standardizing colons and removing introductory hyphens in some cases. No major content changes.

### CODE REVIEW

File: company/about-mattermost/mindsets.md [hunk 1]
```diff
@@ -20,37 +20,41 @@ In this context, here are mindsets for Mattermost:
 
 The best teachers can speed colleagues to surpass them. This mindset helps us constantly grow and rotate into new roles, while preventing single-points of failure where only one person is qualified for a certain task.
 
-### Slow is smooth, smooth is fast
+### Slow is Smooth, Smooth is Fast
```
Comment (jasonblais) [inline company/about-mattermost/mindsets.md:23]: Why are these subheadings in title case?

File: company/about-mattermost/mindsets.md [hunk 2]
```diff
@@ -20,37 +20,41 @@ In this context, here are mindsets for Mattermost:
...
-When facing a high impact problem without a simple and obvious solution, focus your time on 1\) understanding the problem you're solving, 2\) understanding how others have solved the same problem in an enduring way, 3\) what vital changes, if any, are needed to apply a proven solution that works.   
-  
-More: Read GitLab's boring solutions mindset and Dan McKinley's blog post on [Choose boring technology](https://mcfunley.com/choose-boring-technology). 
+When facing a high impact problem without a simple and obvious solution, focus your time on:
+
+* Understanding the problem you're solving.
+* Understanding how others have solved the same problem in an enduring way.
+* What vital changes, if any, are needed to apply a proven solution that works.
+
+**More:** Read GitLab's boring solutions mindset and Dan McKinley's blog post on [Choose boring technology](https://mcfunley.com/choose-boring-technology).
```
Comment (jasonblais) [inline company/about-mattermost/mindsets.md]:
```suggestion
**More:** Read GitLab's boring solutions mindset and Dan McKinley's blog post on [choosing boring technology](https://mcfunley.com/choose-boring-technology).
```

**Review by jasonblais — COMMENTED** [review]
One suggestion and a question.

File: company/about-mattermost/mindsets.md (hunk 1 above)
Comment (justinegeffen) [reply company/about-mattermost/mindsets.md:23]: The rest of the headings are in title case so I wanted to make it consistent here. It's not in line with what's in the style guide but I'm not sure how long it'll take me to change all the headers here - so leaving it as-is for now until I can make a wider change.

**Review by jasonblais — APPROVED** [review]

**Review by natalie-hub — APPROVED** [review]
Thanks Justine! I think this all looks good to merge.

**Pull request merged**